pkg/streaming/commands/kafkaprovider_create.go

```diff
@@ -75,6 +88,23 @@ func (opts *KafkaProviderCreateOptions) Exec(ctx context.Context, c *cli.Config)
 		}
 	}
 	c.Successf("Created kafka provider %q\n", provider.Name)
+	if opts.Tail {
+		err := race.Run(ctx, opts.WaitTimeout,
```

Comment:
do we want to collect logs from the provisioner and gateway deployments as well?